websites/serializers.py

```diff
@@ -263,6 +257,10 @@ def get_draft_url(self, instance):
         """Get the draft url for the site"""
         return instance.get_full_url(version=VERSION_DRAFT)
 
+    def get_has_site_metadata(self, instance):
+        site_metadata = instance.websitecontent_set.filter(type="sitemetadata")
+        return True if (site_metadata and site_metadata.metadata) else False
```

Comment:
I think we need to add this in `WebsiteStatusSerializer`.

I'm not sure if it is required in `WebsiteDetailSerializer`. If it is required in both serializers then I think it would be best to create a mixin, like `WebsiteHasMetadataSerializer`.